### ISSUE TITLE
Cannot find module 'numbro'

### ISSUE THREAD

**Issue by mscreenie** [handsontable/handsontable#3627]
When trying to use a Bare distribution there the error "Cannot find module 'numbro'".

Tried including numbro. Doesn't work.

**Comment by AMBudnik** [comment]
Can you create a demo when we can replicate the issue @mscreenie ?

**Comment by alex-erygin** [comment]
I have same issue

**Comment by budnix** [comment]
@mscreenie, @alex-erygin Did you tried include numbroJS before handsontable scripts? I've created working demo at http://jsfiddle.net/uojf62ja/.

**Comment by alex-erygin** [comment]
@bundix, thank you. This solution works:
```
    <script src="http://docs.handsontable.com/pro/1.5.0/bower_components/numbro/dist/numbro.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.14.1/moment.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/pikaday/1.4.0/pikaday.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/zeroclipboard/2.2.0/ZeroClipboard.min.js"></script>
```

But how can I make it work in a react js component?

**Comment by budnix** [comment]
In react js you should import Handsontable with all required modules via `import`. It should work as described here https://github.com/handsontable/handsontable/issues/3407.

**Comment by mscreenie** [comment]
This is still an issue, just include barebones JS in the order of readme to reproduce.

**Comment by mscreenie** [comment]
I'll demo the issue thanks.

**Comment by budnix** [comment]
Hi @mscreenie, Does this issue with loading Handsontable dependencies still occur? If you use Vanilla JS without any build tool this simple solution should work:
```
<script src="https://cdnjs.cloudflare.com/ajax/libs/numbro/1.9.3/numbro.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.14.1/moment.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/pikaday/1.4.0/pikaday.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/zeroclipboard/2.2.0/ZeroClipboard.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/handsontable/0.28.4/handsontable.min.js"></script>
```
or as one bundle
```
<script src="https://cdnjs.cloudflare.com/ajax/libs/handsontable/0.28.4/handsontable.full.min.js"></script>
```

**Comment by mscreenie** [comment]
@budnix  Thanks for your excellent support. 

I think I've solved this. The problem was not handsontable. I was loading handsontable inside a desktop chrome shell based on Chromium 41 - nwjs 12.3. Upon testing in latest Chrome I did not encounter any issues but when I tested in v.41. I encountered the numbro error. It's not a handsontable - I don't expect handsontable to be tested against Chromium versions that old.

Again thanks for your continued support.
